tests/src/Simple/MultiModule/Library.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+public class MultiModuleLibrary
+{
+    public static int ReturnValue = 50;
```

Comment:
Assigning a value to these no longer tests the interesting scenario (the bases get used from the `.cctor` and therefore would be generated even before your fix). Do you intentionally only want to cover the interesting scenario for the GC static base?

(Also a comment saying it's not okay to add any code or references to this class would be nice. I can see how someone would want to reuse this class for other tests, since it looks like a general purpose class to park things in.)